### PR TITLE
feat(aggregators.final): Add option to disable appending _final

### LINKED ISSUE #15267: feat(aggregators.final): Add new tag instead of appending "_final" to fields
### Use Case

We have Telegraf parsing a binary message that comes in at 50Hz. We want to write the 50Hz data to an Influx bucket that is kept locally on a remote location. We want to take the last values and write them to a bucket that is replicated to a central location. We also want to keep the same data schema. By adding a tag instead of changing the field name, we can route the down sampled values to a different influx output. 

### Expected behavior

The field names are the same, but a tag is added to indicate the metrics have passed through the aggregator.

The extra tag can of course be added using tags.

```
[[aggregators.final]]
  tags = {"?frequency"="low"}
  period = "1s"
  drop_original = false
  rename_fields = false
  output_strategy = "periodic"
```

Input
```
data,my=tag my_field=23i
data,my=tag my_field=24i
data,my=tag my_field=25i
data,my=tag my_field=26i
data,my=tag my_field=27i
data,my=tag my_field=28i
data,my=tag my_field=29i
data,my=tag my_field=30i
```

Output
```
data,my=tag my_field=20i
data,my=tag my_field=21i
data,my=tag my_field=22i
data,my=tag my_field=23i
data,my=tag my_field=24i
data,my=tag my_field=25i
data,my=tag,?frequency=low my_field=25i
data,my=tag my_field=26i
data,my=tag my_field=27i
data,my=tag my_field=28i
data,my=tag my_field=29i
data,my=tag my_field=30i
data,my=tag,?frequency=low my_field=30i
```

### Actual behavior

Input
```
data,my=tag my_field=23i
data,my=tag my_field=24i
data,my=tag my_field=25i
data,my=tag my_field=26i
data,my=tag my_field=27i
data,my=tag my_field=28i
data,my=tag my_field=29i
data,my=tag my_field=30i
```

Output
```
data,my=tag my_field=20i
data,my=tag my_field=21i
data,my=tag my_field=22i
data,my=tag my_field=23i
data,my=tag my_field=24i
data,my=tag my_field=25i
data,my=tag,?frequency=low my_field_final=25i
data,my=tag my_field=26i
data,my=tag my_field=27i
data,my=tag my_field=28i
data,my=tag my_field=29i
data,my=tag my_field=30i
data,my=tag,?frequency=low my_field_final=30i
```

### Additional info

I am willing to create a PR for this myself.

### PR DESCRIPTION
**Pull request by LarsStegman**
## Summary

Changing the data schema is not always wanted. Especially when originals are dropped or the final value needs to be routed to a different output.

## Checklist

- [x] No AI generated code was used in this PR

## Related issues

resolves #15267

### CODE REVIEW

File: plugins/aggregators/final/README.md [hunk 1]
```diff
@@ -32,6 +32,9 @@ See the [CONFIGURATION.md][CONFIGURATION.md] for more details.
   ## aggregator and will not get sent to the output plugins.
   # drop_original = false
 
+  ## If true, _final is added to every field name
+  # rename_fields = true
+
```
Comment (srebhan) [inline plugins/aggregators/final/README.md:37]: How about naming this `keep_original_field_names` with a default of `false`? This way, you don't need to initialize the variable or modify the tests...

**Review by srebhan — COMMENTED** [review]
Looks good, just one suggestion...

**Review by srebhan — APPROVED** [review]
Awesome. Thanks @LarsStegman!

**Comment by LarsStegman** [comment]
@srebhan @powersj @DStrand1 I have to compliment you on the way you maintain this repository and how responsive you are to issues and pull requests. Top notch stuff, people!

**Review by powersj — APPROVED** [review]
> @srebhan @powersj @DStrand1 I have to compliment you on the way you maintain this repository and how responsive you are to issues and pull requests. Top notch stuff, people!

Thank you for the kind words!

**Review by DStrand1 — APPROVED** [review]

**Pull request merged**